src/dotnet/commands/dotnet-install/LocalizableStrings.resx

```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@nguerrera i find the recent code's loc are all in commonLocalizableStrings, i guess we want to reduce the count of dlls, is that true? And do you think we should merge this to commonLocalizableStrings?

Reply:
This won't increase the number of dlls. I don't feel that it is necessary to merge this.

Reply:
To be clear, this was referring to putting the string resources in common vs. adding a .resx for the install command, correct?